src/cfnlint/data/CloudSpecs/af-south-1.json

```diff
@@ -26933,6 +26933,8 @@
         "r5.4xlarge.elasticsearch",
         "r5.large.elasticsearch",
         "r5.xlarge.elasticsearch",
+        "t3.medium.elasticsearch",
```

Comment:
unrelated, [these instance type patches from pricing weren't applied due to caching](https://github.com/aws-cloudformation/cfn-python-lint/pull/1554/) in https://github.com/aws-cloudformation/cfn-python-lint/pull/1715 if these are only getting patched in now